e2e-tests/development-runtime/cypress/integration/gatsby-preview/updating.js

```diff
@@ -32,24 +32,22 @@ describe(`Gatsby Preview (Updating)`, () => {
     const count = 5
     update(count)
 
-    cy.get(`li`)
-      .its(`length`)
-      .should(`be`, count + 1)
+    cy.get(`#fake-data li`).should(`have.length`, count + 1)
```

Comment:
this was acting weirdly - for once - it would resolve `.get` as quickly as it wound find any `li` - and also it would not allow `should('be', foo)` to be actual assertion

Reply:
this was also all wrong because there is double the amount of `li` on the page because `#pinc-data` is also on the same page - this was all weird and certainly this test wasn't actually asserting correctly (it was passing by accident or something)